cwl/src/main/scala/wdl4s/cwl/Cwl.scala

```diff
@@ -13,12 +13,16 @@ sealed trait Cwl {
 }
 
 case class Workflow(
-  cwlVersion: Option[CwlVersion] = None,
-  `class`: Witness.`"Workflow"`.T = "Workflow".narrow,
+  cwlVersion: Option[CwlVersion] = Option(CwlVersion.default),
```

Comment:
I can do that, but wouldn't that be a pain if we ever switch to another version?